libs/books/data-access/src/lib/+state/reading-list.reducer.ts

```diff
@@ -18,40 +18,46 @@ export interface ReadingListPartialState {
 export const readingListAdapter: EntityAdapter<ReadingListItem> = createEntityAdapter<
   ReadingListItem
 >({
-  selectId: item => item.bookId
+  selectId: (item) => item.bookId,
```

Comment:
Please avoid such changes while formatting the file.
Applicable wherever this has happened in this file.

Reply:
Changed back the formattings